commands/stats.js

```diff
@@ -88,22 +88,67 @@ const stats = msg => {
 
 	const getCompUserStats = (error, response, parsedBody) => {
 		const competitiveData = parsedBody.us.stats.competitive;
+		const compGmStats = competitiveData.game_stats;
+		const compOvrStats = competitiveData.overall_stats;
 		const emptyObjectCompChecker = Object.getOwnPropertyNames(competitiveData);
 		if (emptyObjectCompChecker.length === 0 || emptyObjectCompChecker.length === 0) {
 			msg.reply("Sorry. This user has missing data. Boo boo doo de doo.");
 			return;
 		}
-		// Sucrizzle - Playing with Git commits
+
+		// Make sure the user has played some games this comp Season.  Not sure if it would be null or 0 or ???  Taking a guess here
+		if (compOvrStats.games == null || compOvrStats.games == 0) {
```

Comment:
I think it's actually compOvrStats.games_played. 

We can prob test this as I'm pretty sure we have at least one person who has played 0 comp games this season.